.github/workflows/coding-style-check.yml

```diff
@@ -9,20 +9,22 @@ on:
       - '**.h'
       - '**.mm'
       - '**.sh'
+      - '**.py'
   pull_request:
     branches: [ main ]
     paths:
       - '**.cpp'
       - '**.h'
       - '**.mm'
       - '**.sh'
+      - '**.py'
 
 permissions:
   contents: read
 
 jobs:
-  check-coding-style:
-    name: Verify coding style
+  check-cpp-style:
+    name: Verify CPP coding style
```

Comment:
This also checks Objective C - so I'd rather not name it CPP coding style. Any suggestions? Also CPP != C so we should also not use check-c-style.

Reply:
"C-like"?

Reply:
Updated!